src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -230,6 +231,23 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         }
     }
 
+    void MainPage::_Navigate(const Editor::ProfileViewModel& profile)
+    {
+        auto state{ winrt::make_self<ProfilePageNavigationState>(profile, _settingsClone.GlobalSettings().ColorSchemes(), *this) };
+
+        // You cannot delete the Windows Powershell and CMD profiles
+        const auto myGuid{ profile.Guid() };
+        const winrt::guid pwshGuid{ ::Microsoft::Console::Utils::GuidFromString(L"{61c54bbd-c2c6-5271-96e7-009a87ff44bf}") };

```

Comment:
These two can probably be `const static`, so they're only ever initialized _once_.